tests/device_tests/solana/test_address.py

```diff
@@ -0,0 +1,42 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2023 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+import pytest
+
+from trezorlib.debuglink import TrezorClientDebugLink as Client
+from trezorlib.solana import get_address
+from trezorlib.tools import parse_path
+
+from ...common import parametrize_using_common_fixtures
+
+pytestmark = [
+    pytest.mark.altcoin,
+    pytest.mark.solana,
+    pytest.mark.skip_t1,
+]
+
+
+@parametrize_using_common_fixtures(
+    "solana/get_address.json",
+)
+def test_solana_get_address(client: Client, parameters, result):
+    client.init_device(new_session=True)
```

Comment:
why do you call init_device here?

Reply:
I think I copied the tests from Cardano which call `init_device` to set `derive_cardano=True` so I just removed the parameter here instead of removing the whole function. Removed from all three test functions in https://github.com/trezor/trezor-firmware/pull/3359/commits/5cc7eab0731883cd9af6fb2e3eba1398a667f12e#diff-bcf8436b9cb87e3509d72fce300324b5587a11b27b4787d05b40ea3a8ec57ce8L36.